pennylane/_qubit_device.py

```diff
@@ -746,6 +748,7 @@ def statistics(
                 obs.return_type = m.return_type
             else:
                 obs = m
+            # TODO: Remove return_type when `observables` argument is removed from this method
```

Comment:
Can we not do this now? I thought `observables` is a list of measurement processes?

Reply:
Nope. Given that `statistics` is user facing, there might be some users that still use it with a list of observables. That is why we added the deprecation cycle, to tell users that in release `v0.29` this function will only allow the `circuit` argument.

Reply:
I see, ok.